exec/plugin-commands-script-runners/src/regexp-command.ts

```diff
@@ -0,0 +1,15 @@
+export function buildRegExpFromCommand (
+  command: string
+): RegExp | null {
+  if (command.length < 3) {
+    return null
+  }
+  if (command[0] !== '/' || command.lastIndexOf('/') < 1) {
+    return null
+  }
+  try {
+    return new RegExp(command.slice(0, command.lastIndexOf('/')).slice(1))
```

Comment:
so something can be written after the last /? what?

Reply:
same as https://github.com/pnpm/pnpm/pull/5871#discussion_r1089343506